admin/pg12/upgrade-to-pg12

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+docker-compose down
+docker-compose \
+    -f docker-compose.yml \
+    -f admin/pg12/pg-9.5-stopped.yml \
+    up -d
+docker-compose stop indexer search mq redis
+
+docker-compose exec db apt-get update
+docker-compose exec db apt-get install \
+    --no-install-suggests \
+    --no-install-recommends \
+    -y \
+    postgresql-12 \
+    postgresql-client-12 \
+    postgresql-server-dev-12 \
+    sudo
+
+PGDATA=/var/lib/postgresql/data
+docker-compose exec db \
+    sudo -u postgres /usr/lib/postgresql/9.5/bin/pg_ctl start -w -D "$PGDATA"
+sleep 5
+
+CURRENT_PG_VERSION=$(docker-compose exec db psql -U musicbrainz -d musicbrainz_db -tAq -c 'SHOW server_version_num')
+if echo "$CURRENT_PG_VERSION" | grep -v '^905'; then
+    echo "Error: Current postgres version should be < 90600, not $CURRENT_PG_VERSON"
+    exit 1
+fi
+
+docker-compose exec musicbrainz git fetch --depth=1 --update-shallow origin '+refs/heads/pg12:refs/remotes/origin/pg12'
+docker-compose exec musicbrainz git reset --hard origin/pg12
+
+docker-compose exec musicbrainz \
+    sh -c 'eval "$(perl -Mlocal::lib)"; cd admin; ./psql --system READWRITE < sql/updates/20200518-pg12-before-upgrade.sql 2>&1'
+
+docker-compose stop db
+
+docker-compose \
+    -f docker-compose.yml \
+    -f admin/pg12/pg-9.5-stopped.yml \
+    up -d db
+
+docker cp admin/pg12/do-pg_upgrade.sh musicbrainz-docker_db_1:/tmp/
+docker-compose exec db /tmp/do-pg_upgrade.sh
```

Comment:
If it is not intended to be run on Docker host, please commit this script without execution rights. It can be run with:
```suggestion
docker-compose exec db bash /tmp/do-pg_upgrade.sh
```

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-docker/pull/146/commits/c99b967c01d842c004dc2f158742d45494089469